tests/queries/models.py

```diff
@@ -730,3 +731,17 @@ class Meta:
 class CustomDbColumn(models.Model):
     custom_column = models.IntegerField(db_column='custom_name', null=True)
     ip_address = models.GenericIPAddressField(null=True)
+
+
+class CreatedField(models.DateTimeField):
+    def __init__(self, *args, **kwargs):
+        kwargs.setdefault('default', Now)
+        super().__init__(*args, **kwargs)
+
+    @property
+    def db_returning(self):
```

Comment:
Again, this could be a class level attribute.

Reply:
Agreed.